.github/workflows/test_build_deb.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
       uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
       with:
         path: "cloud-android-orchestration"
+    - name: check format
+      run: if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
nit: Could this be a separate file under `.github/workflows/` to validate the formatting and other tests separately?

Reply:
Done
Sorry for the force-push spam, I was trying to reuse the stuff repeated between the workflows, but it seems it's not possible or straighforward.